app/views/invites/_form.html.haml

```diff
@@ -13,5 +13,10 @@
     %p Cette personne aura le droit de modifier votre dossier.
 
   = form_tag dossier_invites_path(dossier), remote: true, method: :post, class: 'form' do
-    = email_field_tag :invite_email, '', class: 'small', placeholder: 'adresse email', required: true
-    = submit_tag 'Envoyer une invitation', class: 'button accepted'
+    .row
+      .col
+        = email_field_tag :invite_email, '', class: 'small', placeholder: 'adresse email', required: true
+      .col
+        = text_area_tag :invite_message, '', class: 'small', placeholder: 'message'
```

Comment:
On peut peut-être mettre un message plus explicite :

 - soit un placeholder du genre "Bonjour, merci de me donner votre avis au sujet de ce dossier"
 - soit le mettre dans le champ value pour systèmatiquement avoir un message

Ceci-dit, j'ai l'impression qu'il n'est pas systèmatiquement nécessaire d'avoir un message donc peut être que le placeholder suffit.

Reply:
Si on met juste un placeholder, on va devoir mettre

> Écrivez ici un message particulier (optionnel)

Mais les instructeurs ne sauront pas le texte qui est envoyé s'ils ne mettent rien dans le champ optionnel.

J'ai l'impression qu'il serait donc mieux de mettre un message par défaut directement en value, qui serait inséré systématiquement dans l'email. Mais je ne vois pas comment formuler un message par défaut…

Du coup allez, partons sur un placeholder `Bonjour, merci de me donner votre avis au sujet de ce dossier (optionnel)`, on itérera après…